### PR TITLE
Stop altering default last post info

### PR DESCRIPTION
**Pull request by Dezash**
Currently, if you disable "Can read forum" permission, this extension will hide last post info even if "Can read topics started by others" is set to Yes. Since I have some forums which cannot be read by guests, I noticed that this extension replaces last post info with "Classified information". I suppose that by design setting "Can read topics started by others" to yes should mean that default behavior is not altered.

Before:
before
After:
after

This PR also fixes incorrectly placed language files which prevented the permissions tab in ACP from working.

### CODE REVIEW

**Comment by brunoais** [comment]
> Currently, if you disable "Can read forum" permission, this extension will hide last post info even if "Can read topics started by others" is set to Yes

That is by design. If you cannot read the forum, you cannot see anything so "Can read topics started by others" becomes useless if you set "Can read forum" to `no`

Are you able to see anything in the forum if you set "Can read forum" to `no`?

**Comment by Dezash** [comment]
By default, if you set "Can read forum" to no, you can still see the last post info, however this behavior is altered by the extension.

File: event/main_listener.php [hunk 1]
```diff
@@ -363,13 +363,7 @@ public function phpbb_viewforum_get_topic_ids_data($event)
 
 	public function phpbb_display_forums_modify_template_vars($event)
 	{
-		
-		$post_access = $this->permission_evaluation->permission_evaluate(array(
-			'forum_id' => $event['row']['forum_id_last_post'],
-			'post_id' => $event['row']['forum_last_post_id'],
-		));
```
Comment (brunoais) [inline event/main_listener.php]: Do not delete permission_evaluate please.

File: event/main_listener.php [hunk 2]
```diff
@@ -363,13 +363,7 @@ public function phpbb_viewforum_get_topic_ids_data($event)
...
-
-		if ($post_access !== accesses::FULL_READ)
+		if (!$this->auth->acl_get('f_read_others_topics_brunoais', $event['row']['forum_id']))
```
Comment (brunoais) [inline event/main_listener.php]: make use of `accesses::NO_READ_OTHER` instead please.

**Review by brunoais — CHANGES_REQUESTED** [review]
Final changes

**Review by brunoais — APPROVED** [review]

**Comment by brunoais** [comment]
Thank you

**Pull request merged**